src/main/java/Lesson7/User.java

```diff
@@ -0,0 +1,99 @@
+package Lesson7;
+
+import java.util.Locale;
+
+public class User {
+    private String name;
+    private String surName;
+    private int currentCash;
+    public User(String name,String surName, int currentCash){
+        setName(name);
+        setSurName(surName);
+        setCurrentCash(currentCash);
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        boolean checker=true;
+        String checked=name.toLowerCase(Locale.ROOT);
+        while(checker){
+            for(int i=0;i<checked.length();i++){
+                    if(name.charAt(i)<97||name.charAt(i)>122){
+                        System.out.println("incorrect character in name");
+                        break;
+                    }
+            }
+            checker=false;
+        }
+        this.name = name;
+
+    }
+
+    public String getSurName() {
+        return surName;
+    }
+
+    public void setSurName(String surName) {
+        boolean checker=true;
+        String checked=surName.toLowerCase(Locale.ROOT);
+        while(checker){
+            for(int i=0;i<checked.length();i++){
+                if(name.charAt(i)<97||name.charAt(i)>122){
+                    System.out.println("incorrect character in Surname");
+                    break;
+                }
+            }
+            checker=false;
+        }
+        this.surName=surName;
+    }
+
+    public int getCurrentCash() {
+        return currentCash;
+    }
+
+    public void setCurrentCash(int currentCash) {
+        if(currentCash<0){
+            currentCash=0;
+        }
```

Comment:
what if the user has an overdraft and the amount should be a negative number

Reply:
ok will think about this case